WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -11,37 +11,37 @@
   <StackPanel Classes="settingsLayout">
 
     <DockPanel>
-      <TextBlock Text="Dark Mode" />
+      <TextBlock Text="Dark mode" />
       <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Start Wasabi with the system" />
+      <TextBlock Text="Run Wasabi when computer starts" />
       <ToggleSwitch IsChecked="{Binding RunOnSystemStartup}" Command="{Binding StartupCommand}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Hide Window on close" />
+      <TextBlock Text="Minimize Wasabi on close" />
```

Comment:
it will actually be hiding the window.. not minimizing it.

We probably should use: "Allow wasabi to run in the background" instead.